src/test/java/it/gov/pagopa/payhub/activities/activity/classifications/IufClassificationActivityTest.java

```diff
@@ -0,0 +1,112 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dao.ClassificationDao;
+import it.gov.pagopa.payhub.activities.dao.PaymentsReportingDao;
+import it.gov.pagopa.payhub.activities.dto.classifications.ClassificationDTO;
+import it.gov.pagopa.payhub.activities.dto.classifications.Transfer2ClassifyDTO;
+import it.gov.pagopa.payhub.activities.dto.classifications.IufClassificationActivityResult;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import it.gov.pagopa.payhub.activities.utility.faker.ClassificationFaker;
+import it.gov.pagopa.payhub.activities.utility.faker.PaymentsReportingFaker;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class IufClassificationActivityTest {
+    @Mock
+    private PaymentsReportingDao paymentsReportingDao;
+
+    @Mock
+    private ClassificationDao classificationDao;
+
+    private IufClassificationActivity iufClassificationActivity;
+
+    private static final Long ORGANIZATIONID = 1L;
+    private static final Long TREASURYID = 1L;
+    private static final String IUF = "IUF";
+
+    @BeforeEach
+    void init() {
+        iufClassificationActivity = new IufClassificationActivityImpl(paymentsReportingDao, classificationDao);
+    }
+
+    @AfterEach
+    void verifyNoMoreInteractions(){
+        Mockito.verifyNoMoreInteractions(paymentsReportingDao);
```

Comment:
let's verify no more interactions on all mocks
```suggestion
        Mockito.verifyNoMoreInteractions(paymentsReportingDao, classificationDao);
```

Reply:
OK